src/firefly/js/ui/DownloadDialog.jsx

```diff
@@ -130,7 +131,8 @@ const newBgKey = () => 'DownloadOptionPanel-' + Date.now();
 export function DownloadOptionPanel ({groupKey='DownloadDialog', cutoutSize, help_id, children, style, title, dlParams,
                                          updateSearchRequest, updateDownloadRequest, validateOnSubmit,
                                          cancelText='Cancel', showZipStructure=true, showEmailNotify=false,
-                                         showFileLocation=true, showTitle=true, ...props}) {
+                                         showFileLocation=true, showTitle=true,
+                                         downloadButtonText='Prepare Download', downloadType='package', ...props}) {
```

Comment:
We could derive `downloadButtonText` and similarly `buttonText` from `downloadType` to avoid redundancy. However, I'm open to whichever approach you prefer.

Reply:
did this for `downloadButtonText` based on your suggestion, left `buttonText` as is, since it's passed to the `DownloadButton` component (as opposed to `DownloadOptionPanel`)